PI-System-Deployment-Samples/OnPrem/README.md

```diff
@@ -0,0 +1,78 @@
+# On-Premise PI Server Install PowerShell Sample
+
+**Version:** 1.0.0
+
+[![Build Status](https://dev.azure.com/osieng/engineering/_apis/build/status/product-readiness/PI-System/Deployment_OnPrem?branchName=master)](https://dev.azure.com/osieng/engineering/_build/latest?definitionId=1640&branchName=master)
+
+This sample uses PowerShell to install Microsoft SQL Server Express, the PI Server including PI Data Archive and PI AF Server, and/or a generic self-extracting PI install kit. The script only installs the packages that are specified by flags, so it can be used to run all three installs or only one.
+
+Developed using PowerShell 5.1
+
+## Requirements
+
+- Powershell 5+
+- Install kits for products to be installed
```

Comment:
Which versions of the install kits work?  Or that we have tested with at least?

Reply:
Added this info just above in the README